src/scss/common/tooltip.scss

```
@@ -0,0 +1,6 @@
+.tooltip-zindex {
```

Instead, I think we should apply this to tooltips in general like this:
```suggestion
div.fade:has(> .tooltip) {
```
This obviously had wider-ranging impact, but all `.fade`s with a `.tooltip` child should be specific enough to reference only BS tooltips.